curriculum/test/test-challenges.js

```diff
@@ -196,16 +196,21 @@ async function setup() {
   }
 
   const meta = {};
+  const fullMeta = {};
   for (const challenge of challenges) {
     const dashedBlockName = challenge.block;
     if (!meta[dashedBlockName]) {
       meta[dashedBlockName] = (
         await getMetaForBlock(dashedBlockName)
       ).challengeOrder;
     }
+    if (!fullMeta[dashedBlockName]) {
+      fullMeta[dashedBlockName] = await getMetaForBlock(dashedBlockName);
+    }
```

Comment:
Rather than having `fullMeta` and `meta`, could we just populate the original `meta` with `getMetaForBlock(dashedBlockName`)?  Then we can pull out `challengeOrder` in the single place it's used

Reply:
Done.

Reply:
Cheers, it looks much cleaner now.